tests/ui/components/search/spellcheckcomponent.js

```diff
@@ -0,0 +1,66 @@
+import DOM from '../../../../src/ui/dom/dom';
+import mockManager from '../../../setup/managermocker';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import SpellCheckComponent from '../../../../src/ui/components/search/spellcheckcomponent';
+
+const globalStorage = {};
+
+const mockCore = {
+  globalStorage: {
+    getState: key => globalStorage[key] || undefined,
+    set: (key, value) => {
+      globalStorage[key] = value;
+    }
+  }
+};
+
+const COMPONENT_MANAGER = mockManager(mockCore);
+
+describe('spellcheck redirect links', () => {
+  let linkUrlParams;
+
+  beforeEach(() => {
```

Comment:
Is it necessary to run this before each test? I kind of like the thought process but not sure if it's necessary for this specific case?

Reply:
ah, ya I can probably remove the beforeEach and just put this in the describe block